src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -225,4 +226,44 @@ describe('Drafts Store', () => {
       assert.deepEqual(store.unsavedAnnotations(), []);
     });
   });
+
+  describe('#deleteNewAndEmptyDrafts2', () => {
+    unroll(
+      '#key',
+      testCase => {
+        store.createDraft(testCase.annotation, testCase.draft);
+        store.deleteNewAndEmptyDrafts([testCase.annotation]);
+        testCase.assertTest(store.getDraft(testCase.annotation));
+      },
+      [
+        {
+          key: 'should remove new and empty drafts',
+          annotation: {
+            id: undefined,
+            $tag: 'my_annotation_tag',
+          },
+          draft: fixtures.emptyDraft,
+          assertTest: assert.isNotOk,
```

Comment:
I think it would be preferable here to make the test case property more obviously meaningful, eg. `shouldRemove: true|false` and then use the corresponding assert in the test body.